cocos/core/assets/texture-cube.jsb.ts

```diff
@@ -132,6 +162,61 @@ Object.defineProperty(textureCubeProto, 'mipmaps', {
     }
 });
 
+Object.defineProperty(textureCubeProto, 'mipmapAtlas', {
+    get () {
+        return this._mipmapAtlas;
+    },
+    set (value) {
+        this._mipmapAtlas = value;
+        if (!this._mipmapAtlas) {
+            this.reset({
+                width: 0,
+                height: 0,
+                mipmapLevel: 0,
+            });
+            return;
+        }
+        const imageAtlasAsset: jsb.ImageAsset = this._mipmapAtlas.atlas.front;
+        if (!imageAtlasAsset.data) {
+            return;
+        }
+        const faceAtlas = this._mipmapAtlas.atlas;
+        const layout = this._mipmapAtlas.layout;
+        const mip0Layout = layout[0];
+
+        const ctx = Object.assign(document.createElement('canvas'), {
+            width: imageAtlasAsset.width,
+            height: imageAtlasAsset.height,
+        }).getContext('2d');
+
+        this.reset({
+            width: mip0Layout.width,
+            height: mip0Layout.height,
+            format: imageAtlasAsset.format,
+            mipmapLevel: layout.length,
+        });
+
+        for (let j = 0; j < layout.length; j++) {
+            const layoutInfo = layout[j];
+            _forEachFace(faceAtlas, (face, faceIndex) => {
+                ctx!.clearRect(0, 0, imageAtlasAsset.width, imageAtlasAsset.height);
+                const drawImg = face.data as HTMLImageElement;
+                ctx!.drawImage(drawImg, 0, 0);
```

Comment:
Do native platforms support drawImage, getImageData?

Reply:
It may not be supported, so remove it for now and use the 'setmipmapAtlas' function to re-implement it on the native platform